config/locales/en.yml

```diff
@@ -2643,6 +2662,44 @@ See the %{link} to find out more about %{sitename}'s features and to start using
     signup:
       start_free_profile: "Start with a free profile, and expand when you're ready!"
 
+  order_management:
+    reports:
+      enterprise_fee_summary:
+        date_end_before_start_error: "must be after start"
+        parameter_not_allowed_error: "You are not authorized to use one or more selected filters for this report."
+        fee_calculated_on_transfer_through_all: "All"
+        fee_type:
+          payment_method: "Payment Transaction"
+          shipping_method: "Shipment"
+        fee_placements:
+          supplier: "Incoming"
+          distributor: "Outgoing"
+          coordinator: "Coordinator"
+        tax_category_name:
+          shipping_instance_rate: "Platform Rate"
+        formats:
+          csv:
+            header:
+              fee_type: "Fee Type"
+              enterprise_name: "Enterprise Owner"
+              fee_name: "Fee Name"
+              customer_name: "Customer"
+              fee_placement: "Fee Placement"
+              fee_calculated_on_transfer_through_name: "Fee Calc on Transfer Through"
+              tax_category_name: "Tax Category"
+              total_amount: "$$ SUM"
+          html:
+            header:
+              fee_type: "Fee Type"
+              enterprise_name: "Enterprise Owner"
+              fee_name: "Fee Name"
+              customer_name: "Customer"
+              fee_placement: "Fee Placement"
+              fee_calculated_on_transfer_through_name: "Fee Calc on Transfer Through"
+              tax_category_name: "Tax Category"
+              total_amount: "$$ SUM"
+        invalid_filter_parameters: "The filters you selected for this report are invalid."
+
```

Comment:
agree but... one step at a time :-)

Reply:
I just say so because Rails engines are almost regular Rails apps and they also have a `config/locale/`

> or locales, simply place the locale files in the config/locales directory, just like you would in an application.

Reply:
@sauloperez Yes, it would be nice, but we would still have to see how to work the Transifex integration for that.

We can start by moving the translations needed for the `order_management` engine to the `order_management` and `js.order_management` I18n scopes, though. I will do this.

Reply:
oh, interesting. we need to make lazyloading work inside engines.
for example, will t('.new_order') in engines/order_management/app/views/order_management/orders/new.html.haml work as  order_management.orders.new.new_order = "New Order" in en.yml?

Reply:
I realized that only the translations for the JS, views, and controller are not yet in the "order_management" I18n scope, but I don't plan to move those in the engine yet because that would be quite complicated. I will just create a tech debt issue for moving these to the engine.

Translations for the services are already in "order_management", so I'm marking this resolved already. :slightly_smiling_face: But if you have objections, feel free to undo.